easybuild/easyconfigs/s/Salmon/Salmon-0.11.2-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'CMakeMake'
+
+name = 'Salmon'
+version = '0.11.2'
+versionsuffix = '-Python-2.7.14'
+
+toolchainopts = {'cstd': 'c++14'}
```

Comment:
@HPC-UniOldenburg This should be merged into the `toolchainopts` line below (or just be removed, it doesn't do anything since this line is basically overwritten by the one below)